code/datums/mutations/passive.dm

```diff
@@ -11,3 +11,19 @@
 /datum/mutation/human/biotechcompat/on_losing(mob/living/carbon/human/owner)
 	owner.adjust_skillchip_complexity_modifier(-1)
 	return ..()
+
+/datum/mutation/human/clever
+	name = "Clever"
+	desc = "Causes the subject to feel just a little bit smarter."
+	quality = POSITIVE
+	instability = 5
```

Comment:
for something meant to give a massive boost to monkeys, this really doesn't give the 'good gene' drawback of a lot of instability. I expected at least 20 imo.

Reply:
Yeah, that seems fair enough. I felt that since it's something that's basically never done, a big power boost to it isn't really a big deal. Just as a raw utility boost for what it does (even if it doesn't do anything most of the time), it is pretty good. I'll up the instability some.